SSI/Imputation/SSI_imputation.ipynb

```diff
@@ -4,18 +4,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This script impute Supplemental Security Income (SSI) recipients and dollar benefit amount to match the aggregates with Social Security Administration (SSA) statistics for SSI. In this current version, we used 2014 CPS data and SSA [2014 annual reports on SSI](https://www.ssa.gov/policy/docs/statcomps/ssi_asr/). Please refer to the documentation in the same folder for more details on methodology and assumptions. The output this script is a personal level dataset that contains CPS personal level ID (PERIDNUM), individual participation indicator (SSI_participation, 0 - not a recipient, 1 - current recipient on file, 2 - imputed recipient), and benefit amount.\n",
+    "This script imputes Supplemental Security Income (SSI) recipients and dollar benefit amount to match the aggregates with Social Security Administration (SSA) statistics for SSI. In this current version, we used 2015 CPS data and SSA [2014 annual reports on SSI](https://www.ssa.gov/policy/docs/statcomps/ssi_asr/). Please refer to the documentation in the same folder for more details on methodology and assumptions. The output this script is a personal level dataset that contains CPS personal level ID (PERIDNUM), individual participation indicator (SSI_participation, 0 - not a recipient, 1 - current recipient on file, 2 - imputed recipient), and benefit amount.\n",
     "\n",
-    "Input: 2014 CPS (cpsmar2014t.csv), 2014 number of recipients by age and state in Dec 2014 (SSI_by_age_state.csv)\n",
+    "Input: 2015 CPS (cpsmar2015t.csv), 2015 number of recipients by age and state in Dec 2015 (SSI_by_age_state.csv)\n",
```

Comment:
So the numbers in SSI_by_age_state.csv is pulled from 2014 annual report, so it should remain 2014 as well. Sorry if this is confusing, but the reason why we're replacing 2014 CPS with 2015 is that 2014 CPS contains 2013 income data, which is not the best option for 2014 administrative data. So we switch to 2015 CPS, keeping all the 2014 target the same.